cocotb/bus.py

```diff
@@ -139,7 +137,7 @@ def capture(self):
 
         Returns:
             dict: A dictionary that supports access by attribute, 
-                where each attribute corresponds to each signal's value.
+            where each attribute corresponds to each signal's value.
```

Comment:
I actually think the indentation is important here. If I'm not mistaken this would render the modified line as being outside the `dict` description.  Can you confirm this?

Reply:
I initially had it like that, but turns out this is a bit of a special case here where the indentation would start a new indented paragraph in the output. It's also mentioned in the Sphinx Google style doc.